CHANGELOG.md

```diff
@@ -35,6 +35,41 @@ Branch: `nilrt/master/next`
 - [Fixed](https://github.com/ni/meta-nilrt/pull/676) a parsing error in the recovery media provisioning scripts.
 
 
+----
+## 10.3
+Branch: `nilrt/24.3/kirkstone`
+
+### nilrt
+#### Added
+- [Added](https://github.com/ni/nilrt/pull/269) libncurses-dev package to the required host dependencies in READEM.kernel.md.
+
+#### Changed
+- [Changed](https://github.com/ni/nilrt/pull/276) a command in README.md to rename the run-mode base image.
+
+#### Removed
+- [Removed](https://github.com/ni/nilrt/pull/274) feed-changelogs, since we don't yet have a good usecase for their inclusion, and they have not proven to be useful.
+
+### meta-nilrt
+#### Added
+- [Added](https://github.com/ni/meta-nilrt/pull/690/) support for getting full disk image.
+- [Added](https://github.com/ni/meta-nilrt/pull/694) setall functionality and misc.
+- [Added](https://github.com/ni/meta-nilrt/pull/698) grub entries and improved set image handling.
+- [Added](https://github.com/ni/meta-nilrt/pull/703) support for custom images, improved error handling and UI.
+- [Added](https://github.com/ni/meta-nilrt/pull/704) compatibilty check, support for answers file, progress bar.
+
+#### Changed
+- [Changed](https://github.com/ni/meta-nilrt/pull/683) DISTRO_VERSION, NILRT_FEED_NAME in nilrt.conf.
+- [Changed](https://github.com/ni/meta-nilrt/pull/688) test to run from a developer machine.
+- [Changed](https://github.com/ni/meta-nilrt/pull/689) test to display current manifest details in all cases.
+- [Fixed](https://github.com/ni/meta-nilrt/pull/691) run-time error caused by missing parameter.
+- [Changed](https://github.com/ni/meta-nilrt/pull/692) test to ensure recent manifest is not earlier than basis manifest.
+- [Changed](https://github.com/ni/meta-nilrt/pull/693) test to make diff hash stable despite stripped versions.
+- [Renamed](https://github.com/ni/meta-nilrt/pull/695) command line args to specify manifests.
+- [Improved](https://github.com/ni/meta-nilrt/pull/696) logging output.
+- [Changed](https://github.com/ni/meta-nilrt/pull/697) test to pass dynamic command-line arguments.
+- [Changed](https://github.com/ni/meta-nilrt/pull/702) the installation of influxdb in the Dockerfile.
```

Comment:
These are all testing changes that don't need to be included.

Reply:
Removed testing changes.